docker-compose.yml

```diff
@@ -135,10 +119,246 @@ services:
     volumes:
       - ./rabbitmq/consumers/stock:/HOME/FLASK-APP
 
+  ## Payment routers
+  payment_router01:
+    image: mongo:6.0.1
+    container_name: payment_router-01
+    command: mongos --port 27017 --configdb payment_rs-config-server/payment_configsvr01:27017,payment_configsvr02:27017,payment_configsvr03:27017 --bind_ip_all
+    ports:
+      - 27017:27017
+    restart: always
+    volumes:
+      - ./payment/scripts:/payment/scripts
+      - payment_mongodb_cluster_router01_db:/data/db
+      - payment_mongodb_cluster_router01_config:/data/configdb
+  payment_router02:
+    image: mongo:6.0.1
+    container_name: payment_router-02
+    command: mongos --port 27017 --configdb payment_rs-config-server/payment_configsvr01:27017,payment_configsvr02:27017,payment_configsvr03:27017 --bind_ip_all
+    volumes:
+      - ./payment/scripts:/payment/scripts
+      - payment_mongodb_cluster_router02_db:/data/db
+      - payment_mongodb_cluster_router02_config:/data/configdb
+    ports:
+      - 27118:27017
+    restart: always
+    links:
+      - payment_router01
+
+  ## Payment config Servers
+  payment_configsvr01:
+    image: mongo:6.0.1
+    container_name: payment_mongo-config-01
+    command: mongod --port 27017 --configsvr --replSet  payment_rs-config-server
+    volumes:
+      - ./payment/scripts:/payment/scripts
+      - payment_mongodb_cluster_configsvr01_db:/data/db
+      - payment_mongodb_cluster_configsvr01_config:/data/configdb
+    ports:
+      - 27119:27017
+    restart: always
+    links:
+      - payment_shard01-a
+
+      - payment_shard02-a
+
```

Comment:
I did not meant the line like this 🙃. The way I meant was between the different docker components. In the current file it is between   '- payment_shard03-a' and 'payment_configsvr02:'for example. And then for all the configservers :)